### DISCUSSION TITLE
Next Js Image Zoom Side By Side

### DISCUSSION THREAD

**Discussion by kevinvasoya** [vercel/next.js#55567] [Help]
### Summary

Hi, We want to add a feature called side-by-side image zoom for our Project which is in Next.js 12 and Typescript. Whenever I add some package it gives an error of dev dependencies regarding type. Can anyone suggest a better package to Zoom in side-by-side images? 
Type Error When I added devDependencies

`RecommendationProduct' cannot be used as a JSX component.
  Its type 'ComponentType<Props>' is not a valid JSX element type.
    Type 'ComponentClass<Props, any>' is not assignable to type 'ElementType'.
      Type 'ComponentClass<Props, any>' is not assignable to type 'new (props: any, deprecatedLegacyContext?: any) => Component<any, any, any>'.
        Construct signature return types 'Component<Props, any, any>' and 'Component<any, any, any>' are incompatible.
          The types returned by 'render()' are incompatible between these types.
            Type 'React.ReactNode' is not assignable to type 'import("c:/Graham&Brown/GrahamBrown/node_modules/@types/react-image-magnifiers/node_modules/@types/react/index").ReactNode'.`

**Comment by icyJoseph** [comment]
Ah, apparently the original `react-image-magnifiers` is somewhat abandoned, and hasn't updated to latest types from `@types/react`, not a Next.js issue.

There's someone who claims to have a functioning fork: 

- https://github.com/vanyapr/react-image-magnifiers 

Install it with: `npm i @vanyapr/react-image-magnifiers`
